core-processor/src/ext.rs

```diff
@@ -400,14 +386,38 @@ impl Ext {
         }
         Ok(())
     }
+
+    fn charge_gas_if_enough(
```

Comment:
can't find the reason this to be inside Ext impl without self param

Reply:
This method has been made "static" on purpose as I can't use `&mut self` in the closure on line 882. I can use just `GasCounter` and `GasAllowanceCounter` from the `self`.

Reply:
I'm exactly about static usage: why not just fn somewhere around but nvm